dev-docs/publisher-api-reference.md

```diff
@@ -549,33 +549,151 @@ Request bids. When `adUnits` or `adUnitCodes` are not specified, request bids fo
+|----------+----------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bidder` | Required | String        | Bidder code. Find the [complete reference for all supported bidders here](bidders.html).                                                                                          |
+| `params` | Required | Object        | Bidder's preferred way of identifying a bid request. Find the [complete reference for all supported bidders here](bidders.html).                                                  |
+| `labels` | Optional | Array<String> | Used for showing responsive ads.  Works with the `sizeConfig` object passed in to [pbjs.setConfig]({{site.baseurl}}/dev-docs/publisher-api-reference.html#module_pbjs.setConfig). |
+
+<a name="addAdUnits-MediaTypes" />
 
-**bid**
+##### Media Types
+
+See the table below for the list of properties in the `mediaTypes` object of the ad unit.  For example ad units showing the different media types, see the [Examples](#addAdUnits-Examples) below.
 
 {: .table .table-bordered .table-striped }
-| Name     | Scope     | Type          | Description                                                                                                                                                                                                  |
-| :----    | :-------- | :-------      | :-----------                                                                                                                                                                                                 |
-| `bidder` | required  | string        | The bidder code. Find the [complete list here](bidders.html).                                                                                                                                                |
-| `params` | required  | object        | The bidder's preferred way of identifying a bid request. Find the [complete reference here](bidders.html).                                                                                                   |
-| `labels` | optional  | array<string> | An array of string labels, used for showing responsive ads.  Works with the `sizeConfig` object passed in to [pbjs.setConfig]({{site.baseurl}}/dev-docs/publisher-api-reference.html#module_pbjs.setConfig). |
+| Name     | Scope                                 | Type   | Description                                                                                                        |
+|----------+---------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------|
+| `native` | Required, unless `video` is present.  | Object | Defines properties of a native ad.  For an example native ad unit, see [the native example below](#adUnit-native). |
```

Comment:
What about mediaTypes.banner?  Do we support this today?

Reply:
Good catch - got it figured it out via another comment by Kendall.